apps/cow-fi/services/cms/index.ts

```
@@ -146,9 +148,121 @@ export async function getArticles({
+  const trimmedSearchTerm = searchTerm.trim()
+
+  if (!trimmedSearchTerm) {
+    return { data: [], meta: { pagination: { page, pageSize, pageCount: 0, total: 0 } } }
+  }
+
+  try {
+    // 1. Log the raw search term
+    console.log('Searching for:', JSON.stringify(trimmedSearchTerm))
+
+    // 2. Explicit filter structure with indices
+    const filters = {
```

ESLint

[error] 'filters' is assigned a value but never used. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)